detect_secrets/plugins/password.py

```diff
@@ -0,0 +1,68 @@
+"""
+This code was extracted in part from
+https://github.com/PyCQA/bandit. Using similar heuristic logic,
```

Comment:
Greeting @ericwb and @viraptor, you both seem like the primary maintainers of Bandit.

This code is largely borrowed from [plugins/general_hardcoded_password.py](https://github.com/PyCQA/bandit/blob/master/bandit/plugins/general_hardcoded_password.py#L24) and I've tried my best to reference it as such per the APACHE license of Bandit. Can you please give this your blessing?

Reply:
Thanks @viraptor! Feel free to checkout [PyT](https://github.com/python-security/pyt) sometime ;)